projects/plugins/inspect/rollup.config.js

```diff
@@ -0,0 +1,96 @@
+import alias from '@rollup/plugin-alias';
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+import livereload from 'rollup-plugin-livereload';
+import postcss from 'rollup-plugin-postcss';
+import svelte from 'rollup-plugin-svelte';
+import { terser } from 'rollup-plugin-terser';
```

Comment:
To go along with the switch to `@rollup/plugin-terser`,
```suggestion
import terser from '@rollup/plugin-terser';
```